roles/hosted_engine_setup/README.md

```diff
@@ -43,6 +43,7 @@ Ansible version >= 2.9.21 and < 2.10.0
 | he_force_ip4 | false | Force resolving engine FQDN to ipv4 only using DNS server |
 | he_force_ip6 | false | Force resolving engine FQDN to ipv6 only using DNS server |
 | he_apply_openscap_profile | false | apply a default OpenSCAP security profile on HE VM |
+| he_openscap_profile_name | stig | OpenSCAP profile name, available options: *stig*, *pci-dss*. Requires `he_apply_openscap_profile` to be `True` |
```

Comment:
Hmm, do we want really stig to be the default profile?

@michalskrivanek

Reply:
Currently, users just specify the `he_apply_openscap_profile` bool parameter, and if `true` the default is stig.
So we can keep this behavior by setting `stig` as default (it's only when `he_apply_openscap_profile` is `True` and `he_openscap_profile_name` not specified).

Reply:
OK, makes sense